app/renderer/components/preferences/payment/ledgerBackup.js

```diff
@@ -14,10 +14,12 @@ const globalStyles = require('../../styles/global')
 
 // other
 const aboutActions = require('../../../../../js/about/aboutActions')
+const appActions = require('../../../../../js/actions/appActions')
 
 class LedgerBackupContent extends ImmutableComponent {
   copyToClipboard (text) {
     aboutActions.setClipboard(text)
+    appActions.onLedgerBackupSuccess(true)
```

Comment:
@jasonrsadler this is the only thing that I found, so I will just fix it and then I will merge this PR